api/developerTestAccounts.ts

```diff
@@ -88,6 +89,36 @@ export function fetchDeveloperTestAccountData(
   return axios<DeveloperTestAccount>(reqWithToken);
 }
 
+export function installOauthAppIntoDeveloperTestAccount(
```

Comment:
Open to better name suggestions, but this util is nuanced so I figured it would be best to be hyper specific here to avoid confusion